src/queries/pointsOfInterest.js

```diff
@@ -61,6 +61,17 @@ export const GET_POINTS_OF_INTEREST = gql`
           description
         }
       }
+      openingHours {
+        id
+        sortNumber
+        weekday
+        timeFrom
+        timeTo
+        open
+        dateFrom
+        dateTo
+        description
+      }
```

Comment:
i think neither the `sortNumber` nor the `description` are needed. `id` is in this query as always to avoid caching problems.

i will remove the two unused fields from this query.